### ISSUE TITLE
Empty AccessKey values appeared after refactoring

### ISSUE THREAD

**Issue by ArtemTatarinov** [dotnet/winforms#6760]
* .NET Core Version: 7.0
* Have you experienced this same bug with .NET Framework?: No

After PR #6640 was merged some accessible objects have received the empty `AccessKey` values at the accessibility tools (like `Inspect`). We discussed it with the team and considered it as a regression. Their behavior should be restored as it were before this PR.

**Comment by Tanya-Solyanik** [comment]
```
Olina Zhang (Beyondsoft Corporation):
    
Hi Tanya. Artem, during this private testing, we found .Net 7.0 before PR: [6640](https://github.com/dotnet/winforms/pull/6640) hasn't Accesskey property displayed in Inspect for controls: ComboBox/NumericUpDown/UpDown Spinner/LinkLabel/TrackBar/ListBox/TabControl/TabControlPane/ListView/PropertyGrid/DataGridView, after that PR merged, they has, so we consider it is a regression issue. But we checked they have that AccessKey display in .Net 6.0 and .Net framework. 


```

Since AccessKeys were present in the .NET Framework, and this is not a MAS violation, let's keep that behavior.
